docs/concepts/aa-x-lit.md

```diff
@@ -0,0 +1,68 @@
+# Account Abstraction x Lit MPC
+
+Lit leverages [multi-party computation (MPC)](https://www.wikiwand.com/en/Secure_multi-party_computation?ref=spark.litprotocol.com) to provide a distributed key management system that enables secure reading and writing of data between blockchains and off-chain platforms. With the added benefits of encryption, access control, and programmatic signing, Lit enhances the capabilities of what’s possible with account abstraction.
+
+Section Overview
+
+- Use cases of account abstraction with Lit
+- An intro to account abstraction and multiparty computation (MPC)
+- How to use Lit Actions & Programmable Key Pairs (PKPs) to work with AA
+
+## How to use Lit with Account Abstraction (AA)
```

Comment:
wondering if this section could be moved closer to or combined with "How to Utilize AA with Lit"